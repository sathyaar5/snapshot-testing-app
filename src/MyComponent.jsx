import React from 'react';

const MyComponent = () => (
  <div>
    <h1>Hello, Sathya!</h1>
  </div>
);

export default MyComponent;
