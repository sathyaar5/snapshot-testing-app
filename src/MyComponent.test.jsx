import React from 'react';
import renderer from 'react-test-renderer';
import MyComponent from './MyComponent';

test('MyComponent renders correctly', () => {
  const tree = renderer
    .create(<MyComponent />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
