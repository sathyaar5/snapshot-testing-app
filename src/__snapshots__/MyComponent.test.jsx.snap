// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent renders correctly 1`] = `
<div>
  <h1>
    Hello, Sathya!
  </h1>
</div>
`;
